src/layouts/EditHomepage/HomepagePreview.tsx

```diff
@@ -175,6 +177,9 @@ export const HomepagePreview = ({
                 subtitle={section.announcements.subtitle}
                 announcementItems={section.announcements.announcement_items}
                 sectionIndex={sectionIndex}
+                announcementScrollRefs={
+                  announcementScrollRefs as Ref<HTMLDivElement>[]
+                }
```

Comment:
not an issue iwth this now but i just realised that instead of casting, we can just declare the type since we alr know it

Reply:
fair 2679320452e4a29dc282d5845fd0001eb0c4b5c2

Reply:
2679320452e4a29dc282d5845fd0001eb0c4b5c2